packages/tools/src/tools/segmentation/strategies/fillCircle.ts

```diff
@@ -96,29 +94,15 @@ function createPointInEllipse(worldInfo: {
   const yRadius = Math.abs(topLeftWorld[1] - bottomRightWorld[1]) / 2;
   const zRadius = Math.abs(topLeftWorld[2] - bottomRightWorld[2]) / 2;
 
-  const radius = Math.max(xRadius, yRadius, zRadius);
-  if (
-    nearOrZero(xRadius, radius) &&
-    nearOrZero(yRadius, radius) &&
-    nearOrZero(zRadius, radius)
-  ) {
-    const sphereObj = {
-      center,
-      radius,
-      radius2: radius * radius,
-    };
-    return (pointLPS) => pointInSphere(sphereObj, pointLPS);
-  }
-  // using circle as a form of ellipse
+  // using ellipse as a form of circle
```

Comment:
I checked the performance of this using the sphere tool and it still seems to be fairly reasonable, so I think we are probably ok using just the ellipse version of it.